diff --git a/openshift/ci-operator/knative-images/kn/Dockerfile b/openshift/ci-operator/knative-images/kn/Dockerfile
index 5c338be8a..710d49182 100755
--- a/openshift/ci-operator/knative-images/kn/Dockerfile
+++ b/openshift/ci-operator/knative-images/kn/Dockerfile
@@ -1,6 +1,8 @@
 # DO NOT EDIT! Generated Dockerfile for cmd/kn.
 ARG GO_BUILDER=registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17
 ARG GO_RUNTIME=registry.access.redhat.com/ubi8/ubi-minimal
+ARG IMAGE_KN_PLUGIN_FUNC_UTIL=registry.redhat.io/openshift-serverless-1/kn-plugin-func-func-util-rhel8@sha256:134a07b2ecd8bb485d7f20bf23e863f5cec5620134c4a26b65e31d23c3076175
+ARG IMAGE_KN_PLUGIN_EVENT_SENDER=registry.redhat.io/openshift-serverless-1/kn-plugin-event-sender-rhel8@sha256:e2ea8370284a4807712849a0ec8c786bbf7cb6cd78dc2309c14e87470df235e6
 
 FROM $GO_BUILDER as builder
 
@@ -9,6 +11,8 @@ COPY . .
 
 ENV CGO_ENABLED=1
 ENV GOEXPERIMENT=strictfipsruntime
+ENV KN_PLUGIN_FUNC_UTIL_IMAGE=$IMAGE_KN_PLUGIN_FUNC_UTIL
+ENV KN_PLUGIN_EVENT_SENDER_IMAGE=$IMAGE_KN_PLUGIN_EVENT_SENDER
 
 RUN go build -tags strictfipsruntime -o /usr/bin/main ./cmd/kn
 
